.github/workflows/ci_cd.yml

```
@@ -36,16 +36,10 @@ jobs:
       api: ${{ steps.paths-filter.outputs.api }}
       ingestion_server: ${{ steps.paths-filter.outputs.ingestion_server }}
       frontend: ${{ steps.paths-filter.outputs.frontend }}
-
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - id: paths-filter
-        name: Check changed paths
-        uses: dorny/paths-filter@v2
-        with:
-          filters: ./.github/filters.yml
+      - uses: actions/checkout@v3
```

nit, sorry: Jobs should have names.

```suggestion
      - name: Checkout repository
        uses: actions/checkout@v3

```